gradle/libs.versions.toml

```diff
@@ -0,0 +1,44 @@
+[versions]
+hypertrace-serviceFramework = "0.1.43"
```

Comment:
might be worth waiting for the couple pending framework prs

Reply:
yeah, that was the plan